backend/src/apiserver/storage/run_store_test_setup.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/kubeflow/pipelines/backend/src/apiserver/model"
+	"github.com/testcontainers/testcontainers-go/modules/mysql"
+)
+
+type RuneStoreMysqlTestEnv struct {
```

Comment:
most of the implementation here feels pretty generic and re-usable, does it need to be scoped to Runs? it seems like if we update `cleanTables` to clean out all the tables [here](https://github.com/kubeflow/pipelines/pull/11889/files#diff-210f8bf0fba8e19d865e643c3c85bfb489835c6d13e60a1f42918b758fab485fR47), it seems - a lot of the logic can be generalized to be used for the other store tests no?

Reply:
right, but it seems like we're going to be repeating a lot of this logic when we add this for the other stores no? can we not just create a `test_util` package in `backend/src/apiserver/storage`, and make this re-usable for all the stores? from what I can tell the only difference here are the tables, we could just extend it to support all tables right [[1]](https://github.com/kubeflow/pipelines/blob/375226aed17be43898578a9b6ab9d6e5b02ff98e/backend/src/apiserver/storage/db_fake.go#L47)?